Module4_Labs/Lab3_File_System/Cards/4.md

```diff
@@ -46,7 +46,7 @@ Your `print_ordered_file_structure()` should write the following to the output f
 ...
 ```
 
-In order to achieve this, you're going to want to implement a `DFS` style traversal. However, in this instance, in order to make sure the files are in the correct relative order, you need to use a `stack` instead of a `queue`. 
+In order to achieve this, you're going to want to implement a `DFS` style traversal, using a `stack`
 
```

Comment:
I would keep the part about why you need a stack, maybe just reword it or you can simply prompt the student to think about why we are using a stack in this instance